client/my-sites/upgrades/domain-management/name-servers/dns-template-button.jsx

```diff
@@ -1,14 +1,14 @@
 /**
  * External dependencies
  */
-import React, { Component } from 'react';
+import React from 'react';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 
-class DnsTemplateButton extends Component {
+class DnsTemplateButton extends Button {
```

Comment:
Yes, as soon as you say that the `ButtonGroup` should be a component, it seems obvious to me. For some reason, I didn't think of that before.